pagopa/mui-italia
# feat(PN-9572): adaptation to standards WCAG 2.2 about target size

## Short description
Expand target size of Button and Link to minimum 24x24px.

### Preview
![image](https://github.com/pagopa/mui-italia/assets/73442810/b2540266-4ca8-4e5b-ac9e-f6598819820a)
![image](https://github.com/pagopa/mui-italia/assets/73442810/ab678b50-6619-4239-b3af-ddb558af1948)
![image](https://github.com/pagopa/mui-italia/assets/73442810/a34bdeb9-c207-4da7-a5dc-38ccc70d9264)


## List of changes proposed in this pull request
- Add minHeight and minWidth to MuiButton
- Add minHeight and minWidth to MuiIconButton
- Add minHeight and minWidth to MuiSvgIcon
- Add minHeight and minWidth to MuiLink

## Product
Piattaforma Notifiche -> SEND

## How to test
Run storybook and check that the width and the height is not less than 24px for:
- ButtonNaked
- every link and button in Footer
- every link and button in HeaderAccount (in particular for _Logged In With Dropdown_)
- Inputs > IconButton
